### DISCUSSION TITLE
Approaching React Financial Charts for the First Time

### DISCUSSION THREAD

**Discussion by steinitz** [reactivemarkets/react-financial-charts#561] [General]
Perusing the React Financial Charts code gives the impression of a mighty piece of engineering.  Bravo and thank you.

But for a first-time user, well for me anyway, it seems unclear how to add it to an existing project.  An intrepid explorer on [Stack Overflow](https://stackoverflow.com/questions/68483822/how-to-can-i-use-react-financial-charts-library-in-reactjs) discovered this [codesandbox](https://codesandbox.io/s/react-financial-charts-demo-c88wz) which demonstrates its use clearly and without clutter.  Kudos to both of them.

Its easy enough to find that in Google but maybe it deserves a place here.

![Screen Shot 2021-08-11 at 12 42 59 pm](https://user-images.githubusercontent.com/42371298/128961433-cea41b83-7557-446d-b1e6-265b778b5752.png)

**Comment by Wapaca** [comment]
Thank you, i was looking for some code example
